### PR TITLE
GitHub OSS Migration Prep

### PR DESCRIPTION
**Pull request by ewega**
This pull request includes several important updates to the repository, primarily focusing on documentation improvements and the addition of new guidelines and templates. The key changes include the addition of a Code of Conduct, contributing guidelines, security policy, support guidelines, and updates to the README for better clarity and structure.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This is a comment
+# Each line is a file pattern followed by one or more owners.
+
+# The '*' pattern refers to all files in the repository.
+* @jasonmoodie @Eldrick19
```
Comment (Eldrick19) [inline .github/CODEOWNERS:5]: Are you alright with both of us maintaining this repo @jasonmoodie?

File: README.md [hunk 2]
```diff
@@ -75,6 +78,10 @@ In order to connect we'll need to generate a token and link to your metrics data
 ## Publishing
 If you need help deploying or publishing this script, please see: [Publish README](/publish/README.md)
 
+## Maintainers
 
+@jasonmoodie, @Eldrick19
```
Comment (Eldrick19) [inline README.md:83]: Another Maintainer check

File: .github/CODEOWNERS (hunk 1 above)
Comment (Eldrick19) [reply .github/CODEOWNERS:5]: If you would like me to be a maintainer can you add me as an admin to the repo?

**Review by Eldrick19 — COMMENTED** [review]
TODOs prior to migrating to GitHub ownership:
- [ ] Add Release automation
- [ ] CI for Power Queries (optional)

**Review by jasonmoodie — APPROVED** [review]
Looks good, thanks for the feedback @Eldrick19.

**Pull request merged**